components/layout_2020/flow/inline.rs

```diff
@@ -596,12 +639,13 @@ impl<'a, 'b> InlineFormattingContextState<'a, 'b> {
 
     /// Given the amount of whitespace trimmed from the line and taking into consideration
     /// the `text-align` property, calculate where the line under construction starts in
-    /// the inline axis.
-    fn calculate_inline_start_for_current_line(
+    /// the inline axis as well as the adjustment needed for every justification opportunity
+    /// to account for `text-align: justify`.
+    fn calculate_current_line_inline_start_and_justification_adjustment(
         &self,
         containing_block: &ContainingBlock,
```

Comment:
Unrelated to this PR, but why have this parameter if it's just `self.containing_block`?

Reply:
Great question. I've gone ahead and fixed this. I'm sure it was just an oversight.